### PR TITLE
OCP-40620 - allow from router and allow from host network policy in upgrade scenario

### PR DESCRIPTION
**Pull request by asood-rh**
All three policies (allow from router- new label, allow from router -old label and allow from hostnetork )are applied respectively to three projects.
The test case ensures the policies do what they are supposed (postive path).
Default deny policies are also applied but no validation around it, just assuming the policies do deny traffic to the project.

Testing:-
vSphere (Hostnetwork)
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3266/console (pre upgrade)
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3267/console (post upgrade)

AWS (LoadBalancer Service)
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3268/console (pre upgrade)
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3269/console

/cc @openshift/team-sdn-qe

### CODE REVIEW

File: features/upgrade/sdn/policy-upgrade.feature [hunk 1]
```diff
@@ -233,3 +233,121 @@ Feature: SDN compoment upgrade testing
...
+  # @case_id OCP-40620
+  @admin
+  @upgrade-check
+  Scenario: Check allow from router and allow from hostnetwork policy are functional post upgrade 
+    Given I switch to cluster admin pseudo user
+    When I use the "policy-upgrade3" project
+    Given a pod becomes ready with labels:
+      | name=test-pods |
+    And I wait up to 60 seconds for a web server to become available via the "test-service" route
+    Then the step should succeed
```
Comment (pruan-rht) [inline features/upgrade/sdn/policy-upgrade.feature:333]: @asood-rh if the previous `And I wait up to 60...` step checks for success for failure, then you don't need to post check `Then the step should succeed`

Comment (pruan-rht) [reply features/upgrade/sdn/policy-upgrade.feature:333]: @asood-rh so for that [step I mentioned ](https://github.com/openshift/verification-tests/blob/master/features/step_definitions/net.rb#L37) above it indeed checks for success/failure, so I think we can remove line 333.  The same argument applies to all other steps that exhibit that kind of behavior.  Even though in cucumber the key words `GIven/And/Then` are treated equally, semantic-wise, steps using `Given` implies the status check is embedded.

Comment (asood-rh) [reply features/upgrade/sdn/policy-upgrade.feature]: @pruan-rht Thanks for review. 
Removed the suggested post checks. Reviewed other post checks as well, they are needed.
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3270/console
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3271/console

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**